bms/bms_app/static/bms_app/css/create_project.css

```diff
@@ -4,9 +4,39 @@
 }
 
 .panel-body {
-    padding: 10px 10px 0px 10px;
+	padding: 10px 10px 0px 10px;
```

Comment:
In shorthand properties 10 px(top) 10px (right) 0px(bottom) 10px (left); if right and left value is equal then no need to mention left value. eg 10px 10px 0; is sufficient,